docs/question.md

```diff
@@ -75,7 +75,7 @@ The `info.json` file for each question defines properties of the question. For e
 | `shareSourcePublicly`    | boolean | Whether the the source code of the question should be available                                                                                                        |
 | `sharingSets`            | array   | Sharing sets which the question belongs to                                                                                                                             |
 
-For details see the [format specification for question `info.json`](https://github.com/PrairieLearn/PrairieLearn/blob/master/apps/prairielearn/src/schemas/schemas/infoQuestion.json)
+!!! info "[`infoQuestion.json` reference](schemas/infoQuestion.md) :robot:"
```

Comment:
Can we avoid showing deprecated properties entirely?

<img width="808" alt="Screenshot 2025-03-27 at 14 19 50" src="https://github.com/user-attachments/assets/c91dfebb-5ac7-41de-8644-08bb5190a1b4" />

Reply:
Good idea -- I'll see if I can add this feature.

Reply:
This is tracked by https://github.com/sbrunner/jsonschema2md/pull/437

Reply:
Resolved in [23c429f](https://github.com/PrairieLearn/PrairieLearn/pull/11615/commits/23c429f4263145c7e9459dfd0836fd1c1d19eb1d).